### ISSUE TITLE
Use the accuracy tested by CoreNLP on the tree nodes as the accuracy of Recursive NN model in 2013 paper

### ISSUE THREAD

**Issue by WadeYin9712** [stanfordnlp/CoreNLP#933]
Hi! Can we use CoreNLP and its results as the performance of Recursive model proposed in your 2013 paper? Thx!

**Comment by AngledLuffa** [comment]
Which model? Sentiment or parser?

**Comment by WadeYin9712** [comment]
Oh, I mean sentiment analysis on all the node. Because somehow I find that the results on phrase node might be much lower than the results reported in that paper.

**Comment by AngledLuffa** [comment]
The original sentiment dataset had quite a few data errors which were
adjusted before training the current model.  It's not surprising that the
trained model wouldn't agree after being trained on some slightly different
data.

**Comment by WadeYin9712** [comment]
Oh, I see. So if we want to report the results of the proposed Recursive NN in that paper, just use the model in current CoreNLP API, right?

**Comment by AngledLuffa** [comment]
I mean, if you want the best results we got on the Sentiment dataset, it's
best to use what was reported in the paper.  We still have that model
collecting dust in a Matlab folder somewhere.

If you want to compare the current sentiment model with a different
dataset, then CoreNLP is the best approach.

John

**Comment by WadeYin9712** [comment]
Actually I want to do some analysis on the phrase nodes in that paper. And because you've adjusted a lot on the 2013 dataset, the performance of current model might be totally different from your reported one. Also, it's not fair to compare our model with your reported one because that one was trained by QUITE A FEW errors. We've tested the CoreNLP model on the test set and find there exists a HUGE performance drop compared with the accuracy you reported. We've implemented several baselines, also with BERT structure, and the performance is still much lower than the 2013 results. This is the reason why we want to use the results produced by CoreNLP.

**Comment by AngledLuffa** [comment]
If you're using the stanford sentiment training set as currently released,
you can also retrain a model on that dataset and hopefully get closer to
the reported accuracy,

John

**Comment by WadeYin9712** [comment]
All right! Thanks a lot for your reply! Would you mind tell how much you adjusted the annotations?

**Comment by AngledLuffa** [comment]
You can look for some of the changes in a tool called

ReadSentimentDataset.java

https://github.com/stanfordnlp/CoreNLP/blob/master/src/edu/stanford/nlp/sentiment/ReadSentimentDataset.java


Actually, I can't remember if the publicly available PTB trees have these
changes already applied.  They might, actually.

**Comment by WadeYin9712** [comment]
Wow, thanks a lot! I'll carefully check it and try to do some retraining experiments. Thanks again.
